src/js/webrequest.js

```diff
@@ -1732,7 +1754,14 @@ function startListeners() {
   chrome.webRequest.onBeforeRequest.addListener(filterWarRequests, {
     urls: chrome.runtime.getManifest().web_accessible_resources.map(
       path => chrome.runtime.getURL(path))
-  }, ["blocking"]);
+  }, ['blocking']);
+
+  if (utils.hasOwn(chrome.webRequest.OnBeforeRequestOptions, 'REQUESTBODY')) {
```

Comment:
We don't register in Chrome because lol it's `REQUEST_BODY` there, but ... Should we have a more robust check? The check needs to be synchronous though, to keep the listener "persistent".